src/Config.php

```diff
@@ -18,6 +18,9 @@ public function getRules() : array {
 			'@PSR2' => true,
 			'align_multiline_comment' => true,
 			'array_indentation' => true,
+			'array_syntax' => [
+				'syntax' => 'short',
```

Comment:
this is the new default already https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0/UPGRADE-v3.md#changed-default-values-of-options